mkdocs_drawio/js/drawio.js

```diff
@@ -0,0 +1,17 @@
+const colorize = async function () {
+    document.querySelectorAll("div.mxgraph").forEach(function (div) {
+        let style = div.getAttribute("style").replace(/color-scheme: light;/gi, "");
+        div.setAttribute("style", style);
+    });
+};
+
+if (typeof document$ !== "undefined" && typeof document$.subscribe === "function") {
+    document$.subscribe(({ body }) => {
+        GraphViewer.processElements()
+        colorize();
+      });
+} else {
```

Comment:
This will automatically back in support for the instant reload of material themes. Therefore the explanation for how to manually add it should be removed from the README.

Furthermore, I would like to have a comment explaining the necessity of this if-else block. Because a couple of weeks or months down the round no one will remember why this code exists.